backend/app/treasury/tests/test_covalent.py

```diff
@@ -325,3 +326,79 @@ async def test_get_treasury_success(monkeypatch: MonkeyPatch):
             )
         ],
     )
+
```

Comment:
Please just move these new tests to a separate test file to prevent conflicts with my upcoming branch.